.travis.yml

```diff
@@ -1,4 +1,4 @@
-sudo: false
+sudo: required
```

Comment:
So I'm not happy with these changes :/ We need sudo for 2 things here:
* Install avahi libs + daemon (those are not in the Travis apt plugin white list);
* Run foxbox with root privileges to be able to update host name.